wordpress-mobile/WordPress-Android
# Site Creation: Duplicate reused layouts

When we copied the site creation flow in #8515 to leave the current flow untouched we didn't duplicate the layouts which means if we make a change to the layout it'll change the existing flow as well. I've duplicated the domain layouts as well as the site creation activity layout. I believe these are the only ones we'll use in the new site creation flow, but if I have missed something, this is a good PR to add it.

I've not made any changes to the layouts except for reformatting them. That'll let me reformat the layouts as I work on them without affecting the changeset size.

To test:
* Enable `NEW_SITE_CREATION_ENABLED` flag
* Go to new site creation flow and verify that it's working the same as in `feature/master-site-creation`, specifically the domain screen

P.S: @malinajirka We need to be extra careful about not changing any existing resources for the current site creation flow including the dimensions and strings. I've done a pass at the current changes and I think we are fine, but it's good to keep this in mind as we work on the rest of the changes.
